src/Dependencies/Installer.php

```diff
@@ -39,6 +46,23 @@ public function install()
         return \Result\Result::ok();
     }
 
+    private function installSingle($dep)
+    {
+        //Will only get here if $dep is valid format and matches an entry in whippet.json
+        $type = explode('/', $dep)[0];
+        $name = explode('/', $dep)[1];
+
+        foreach ($this->lockFile->getDependencies($type) as $dep) {
```

Comment:
Again, this looks like it's mostly duplicating `installAll()` - could we extract the common functionality into a function which accepts an array: that way if any of this changes then we only need to update one place rather than two (and risk getting out of sync).